docs/reference/setup/install/deb.asciidoc

```diff
@@ -2,9 +2,15 @@
 === Install Elasticsearch with Debian Package
 
 The Debian package for Elasticsearch can be <<install-deb,downloaded from our website>>
-or from our  <<deb-repo,APT repository>>. It can be used to install
+or from our <<deb-repo,APT repository>>. It can be used to install
 Elasticsearch on any Debian-based system such as Debian and Ubuntu.
 
+This package contains both open source and commercial features. 
```

Comment:
I wonder if we can make it more clear that it's free? The term "commercial features" isn't wrong - the default distribution contains features that aren't under and OSI-approved license - but this phrasing doesn't  doesn't send the message that "it's totally free to use, but isn't totally Apache 2.0 licensed." I still struggle with the best way to phrase it here though.